tools/scripts/vsc_openocd_debug.mk

```diff
@@ -0,0 +1,30 @@
+#
+#	Makefile to create project dependent configs files for debugging with openocd
+#
+
+# Definitions
+SETTINGS_TEMPLATE	= $(NO-OS)/tools/scripts/templates/template_settings.json
+LAUNCH_TEMPLATE		= $(NO-OS)/tools/scripts/templates/template_launch.json
+SETTINGSJSON 		= $(VSCODE_CFG_DIR)/settings.json
+LAUNCHJSON 			= $(VSCODE_CFG_DIR)/launch.json
+
+# Determine gdb path
+GDB_BIN				= $(shell which gdb)
```

Comment:
I would stick to using the `GDB` existing variable (instead of a new one) which in stm32.mk resolves to arm-none-eabi-gdb. If you have gdb-multiarch instead, use a symlink in your system to point an arm-none-eabi-gdb to gdb-multiarch location, it should work.

Reply:
Sorry for the confusion. I should have squashed the commits to one. This part is outdated. I will squash and resubmit.